app/jsx/components/StatCarouselComp.jsx

```diff
@@ -21,39 +55,45 @@ class CampusCarouselComp extends React.Component {
             {this.props.campusName}
           </h2>
           <div className="o-stat--item c-statcarousel__cell">
-            <a href="">99,999</a> Items
+            <a href={"/search?campuses="+this.props.campusID}>{c.item_count.toLocaleString()}</a> Items
           </div>
+        {c.view_count > 0 &&
           <div className="o-stat--view c-statcarousel__cell">
-            <a href="">999,999</a> Views
+            <b>{c.view_count.toLocaleString()}</b> Views
           </div>
-          <div className="o-stat--passed c-statcarousel__cell">
+        }
+      {/* <div className="o-stat--passed c-statcarousel__cell">
              <a href="">9,999</a> Items since UC <br/> OA Policy passed
-          </div>
+          </div> */}
+        {c.journal_count > 0 &&
           <div className="o-stat--journals c-statcarousel__cell">
-             <a href="">99</a> eScholarship Journals
+             <a href={"/"+this.props.campusID+"/journals"}>{c.journal_count.toLocaleString()}</a> eScholarship Journals
           </div>
+        }
+        {c.oru_count > 0 &&
           <div className="o-stat--units c-statcarousel__cell">
-             <a href="">999</a> Research Units
+             <a href={"/"+this.props.campusID+"/units"}>{c.oru_count.toLocaleString()}</a> Research Units
           </div>
+        }
         </div>
         <div className="c-statcarousel__section">
           <h2 className="c-statcarousel__cell-heading">
             All eScholarship
           </h2>
           <div className="o-stat--item c-statcarousel__cell">
-             <a href="">999,999</a> Items
+             <b>{a.all_item_count.toLocaleString()}</b> Items
```

Comment:
Should I wrap these in conditionals in the off chance the dbCache doesn't get populated? But if the dbCache doesn't get populated, a whole bunch of other stuff will be breaking before getting to this point anyway, yes?

Reply:
The dbCache always gets populated before Sinatra takes the stage and binds its port. No need to wrap these in conditionals.